datacenter/ucp/2.1/guides/architecture.md

```diff
@@ -36,11 +36,10 @@ and stops UCP services, based on whether that node is a
 If the node is a:
 
 * **Manager**: the `ucp-agent` service automatically starts serving all UCP
-components including the UCP web UI and data stores used by UCP. By promoting
-a node to manager, UCP automatically becomes highly available and fault tolerant.
+components including the UCP web UI and data stores used by UCP. The `ucp-agent` accomplishes this by [deploying several containers](#ucp-components-in-manager-nodes) on the node. By promoting a node to manager, UCP automatically becomes highly available and fault tolerant. <br/><br/>
```

Comment:
Can you also remove these `<br/>`? If we need more space between list items we can do it with CSS. Doing this will make it harder to restyle the docs because everything has a random number of line breaks.